source/extensions/common/wasm/wasm.h

```diff
@@ -343,20 +387,41 @@ class Wasm : public Envoy::Server::Wasm,
 
 private:
   friend class Context;
+  static const uint32_t kMetricTypeMask = 0x3;
```

Comment:
nit: could you put Mask between 0x2 and 0x4?
Super nit: it might be confusing if `Mask` is a type...

Reply:
Done (first bit) Yes, it would be confusing... I am open to suggestions.  kMaskForMetricType ?